generator/internal/sidekick/templatedata.go

```diff
@@ -213,10 +233,11 @@ func newMessage(m *api.Message, c language.Codec, state *api.APIState) *Message
 		DocLines:           c.FormatDocComments(m.Documentation, state),
 		IsMap:              m.IsMap,
 		IsPageableResponse: m.IsPageableResponse,
+		ID:                 m.ID,
 	}
 }
 
-func newMethod(m *api.Method, c language.Codec, state *api.APIState) *Method {
+func newMethod(s *api.Service, m *api.Method, c language.Codec, state *api.APIState) *Method {
```

Comment:
Within our model many of the elements have a field Parent. I wonder if any of this code is simplified if api.Method has such a field that ties back to the service.

Reply:
Will do this in #479, the cleanup looked too big for post-review changes.